content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -150,7 +150,11 @@ You will need to download the full Istio release containing the auto-completion
 
 Installing the bash auto-completion file
 
-If you are using bash, the `istioctl` auto-completion file is located in the `tools` directory. To use it, copy the `istioctl.bash` file to your home directory, then add the following line to source the `istioctl` tab completion file from your `.bashrc` file:
+If you are using bash, the `istioctl.bash` auto-completion file is located in the `tools` directory. 
+
+You can also generate it by running `istioctl collateral completion --bash -o .`.
```

Comment:
You should put the snippets in a `{{< text bash >}}` which will allow easier copy/paste.